crates/tauri-plugin-holochain-service/android/src/main/java/com/plugin/holochain_service/HolochainServicePlugin.kt

```diff
@@ -71,7 +66,7 @@ class HolochainServicePlugin(private val activity: Activity): Plugin(activity) {
 
         // Start service
         val intent = Intent()
-        intent.setComponent(ComponentName(this.packageName, this.className))
+        intent.setComponent(ComponentName(this.activity, HolochainService::class.java))
```

Comment:
HolochainService's admin IPC binding can only be called from the same package it was started from, so there is no need to pass in the package name as a parameter.